### ISSUE TITLE
Duplicated Operations in Mutate Requests

### ISSUE THREAD

**Issue by BenRKarl** [googleapis/gapic-generator-python#756]
In the Ads templates, [here](https://github.com/googleapis/gapic-generator-python/blob/master/gapic/ads-templates/%25namespace/%25name/%25version/%25sub/services/%25service/client.py.j2#L379), the below logic is generated for our `operations` parameter, an example from proto can be found here: [`MuateAdGroups`](https://github.com/googleapis/googleapis/blob/master/google/ads/googleads/v6/services/ad_group_service.proto#L53) and [`MutateAdGroupsRequest`](https://github.com/googleapis/googleapis/blob/master/google/ads/googleads/v6/services/ad_group_service.proto#L74)

```py
if customer_id is not None:
    request.customer_id = customer_id
if operations is not None:
    request.operations = operations

if operations:
    request.operations.extend(operations)
```

If `operations` is present, `request.operations` is set, and `request.operations.extend` is _also_ called, causing our operations to be duplicated in any request.

**Comment by software-dov** [comment]
Good catch.

**Comment by software-dov** [comment]
Fixed by #760
